docs/source/configuration/entity-caching.mdx

```diff
@@ -129,9 +128,164 @@ This entry contains an object with the `all` field to affect all subgraph reques
       "data": "key2"
     }
 }
+```
+
+### Entity cache invalidation
+
+When existing cache entries need to be replaced, the router supports a couple of ways for you to invalidate entity cache entries:
+- **Invalidation endpoint** - the router exposes an invalidation endpoint that can receive invalidation requests from any authorized service. This is primarily intended as an alternative to the extensions mechanism described below. For example a subgraph could use it to trigger invalidation events "out of band" from any requests received by the router or a platform operator could use it to invalidate cache entries in response to events which aren't directly related to a router.
+- **Subgraph response extensions** - you can send invalidation requests via subgraph response extensions, allowing a subgraph to invalidate cached data right after a mutation.
+
+One invalidation request can invalidate multiple cached entries at once. It can invalidate:
+- All cached entries for a specific subgraph
+- All cached entries for a specific type in a specific subgraph
+- All cached entries for a specific entity in a specific subgraph
+
+To process an invalidation request, the router first sends a `SCAN` command to Redis to find all the keys that match the invalidation request. After iterating over the scan cursor, the router sends a `DEL` command to Redis to remove the matching keys.
+
+#### Configuration
+
+You can configure entity cache invalidation both globally (`preview_entity_cache.invalidation`) and per subgraph (`preview_entity_cache.subgraph.subgraphs.invalidation`).
```

Comment:
I assume that you wouldn't normally want to do both at the same time? or is it expected that you'd usually have a global **and** a per subgraph invalidation? If not, I think I'd prefer that we show two separate yaml examples for global and subgraph invalidations. Makes it a bit more explicit and easier to read.

Reply:
we can have both, and per subgraph is an override of the global config.